doc/whats_new/v1.0.rst

```diff
@@ -73,6 +73,15 @@ Changelog
   Use ``var_`` instead.
   :pr:`18842` by :user:`Hong Shao Yang <hongshaoyang>`.
 
+- |API|: The parameter ``normalize`` of :class:`linear_model.LinearRegression`
+  is deprecated and will be removed in 1.2. The use of
+  :class:`~sklearn.pipeline.Pipeline` with
+  :class:`~sklearn.preprocessing.StandardScaler` is recommended instead.
+  ``normalize`` parameter did not take any effect if ``fit_intercept`` was set
+  to False and therefore was deemed confusing.
+  :pr:`17743` by :user:`Maria Telenczuk <maikia>` and :user:`Guillaume Lemaitre
+  <glemaitre>` and :user:`Alexandre Gramfort <agramfort>`.
```

Comment:
```suggestion
  :pr:`17743` by :user:`Maria Telenczuk <maikia>` and
  :user:`Alexandre Gramfort <agramfort>`.
```

Reply:
You did both most of the job :)

Reply:
As you prefer. But you have input too (remember, during the sprint?)